packages/bits/src/lib/wizard/wizard.component.ts

```diff
@@ -156,8 +156,15 @@ export class WizardComponent implements OnInit, AfterContentInit, AfterViewCheck
         const componentFactory = this.componentFactoryResolver.resolveComponentFactory(WizardStepComponent);
         const componentRef = this.dynamicStep.createComponent(componentFactory);
         const instance: IWizardStepComponent = componentRef.instance;
-        instance.title = wizardStep.title;
-        instance.stepTemplate = wizardStep.stepTemplate;
+        const wizardStepInputs = [
```

Comment:
Doing this has all sorts of side effects and requires maintenance when new properties are added. I'd reconsider the approach here. If you want, we can have a call about this.

Reply:
@lukasholcik I made some changes. Please take a look